common/src/main/java/org/opensearch/ml/common/transport/model_group/MLUpdateModelGroupRequest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.model_group;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.common.io.stream.InputStreamStreamInput;
+import org.opensearch.common.io.stream.OutputStreamStreamOutput;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static org.opensearch.action.ValidateActions.addValidationError;
+
+@Getter
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@ToString
+public class MLUpdateModelGroupRequest extends ActionRequest {
+
+    MLUpdateModelGroupInput updateModelGroupInput;
+
+    @Builder
+    public MLUpdateModelGroupRequest(MLUpdateModelGroupInput updateModelGroupInput) {
+        this.updateModelGroupInput = updateModelGroupInput;
+    }
+
+    public MLUpdateModelGroupRequest(StreamInput in) throws IOException {
+        super(in);
+        this.updateModelGroupInput = new MLUpdateModelGroupInput(in);
+    }
+
+    @Override
+    public ActionRequestValidationException validate() {
+        ActionRequestValidationException exception = null;
+        if (updateModelGroupInput == null) {
+            exception = addValidationError("Update Model group input can't be null", exception);
+        }
+
+        return exception;
```

Comment:
```suggestion
        if (updateModelGroupInput == null)  return addValidationError("Update Model group input can't be null", exception);
```

Reply:
This is something in consistent with all other request classes. Also we might want to return null if updateModelGroupInput != null

Reply:
Then it still can be optimized to:
```
if (updateModelGroupInput == null)  return addValidationError("Update Model group input can't be null", exception);
return null;
```